libs/core/table/directives/table-container.directive.ts

```diff
@@ -0,0 +1,73 @@
+import {
+    ContentChild,
+    Directive,
+    ElementRef,
+    Input,
+    OnDestroy,
+    Renderer2,
+    booleanAttribute,
+    inject
+} from '@angular/core';
+import { ResizeObserverService } from '@fundamental-ngx/cdk/utils';
+import { FD_TOOLBAR } from '@fundamental-ngx/core/toolbar';
+import { Subscription } from 'rxjs';
+import { TableHeaderDirective } from './table-header.directive';
+
+@Directive({
+    selector: '[fdTableContainer], [fd-table-container]',
+    standalone: true
+})
+export class TableContainerDirective implements OnDestroy {
+    /** Whether the scroll is applied on the page and not on the table */
+    @Input({ transform: booleanAttribute })
```

Comment:
The boolean input is probably not necessary if this is only used with this specific edge case of sticky header/scrolling outside container. But removing the input should accompany the change of directive name i mentioned in another comment